railgun/internal/proxy/clientgo_cached_proxy_resolver.go

```diff
@@ -141,6 +219,14 @@ func (p *clientgoCachedProxyResolver) DeleteObject(obj client.Object) error {
 	}
 }
 
+func (p *clientgoCachedProxyResolver) DBMode() string {
+	return p.dbmode
+}
+
+func (p *clientgoCachedProxyResolver) Version() semver.Version {
+	return p.version
+}
+
```

Comment:
Why are these getters necessary? I cannot see any caller of these.

Reply:
Not needed, removed in 36a1a1870a506f5250eca0c815c25535013cc922 :+1: